ynput/ayon-houdini
# Allow to set an explicit AOV identifier for USD render product using attribute `ayon:aov_identifier`

## Changelog Description

Allow to set an explicit AOV identifier for USD render product using attribute `ayon:aov_identifier`

## Additional info

This is a way to override the default behavior that defines what "AOV" identifier is computed for a Render Product. You can now set a string attribute on the Render Product prim called `ayon:aov_identifier`.

For example a Attribute Wrangle LOP could do:
```
s@ayon:aov_identifier = "dcm";
```

So that publish logic will make that render product interpreted as the "dcm" AOV - separating it from other layers.

This may fix (or workaround) https://github.com/ynput/ayon-houdini/issues/105

## Testing notes:

1. Add the attribute to a render product
2. Submit to Deadline
3. Published product should include the AOV identifier.
